tdrs-backend/tdpservice/users/test/test_api/test_users.py

```diff
@@ -52,7 +52,7 @@ def test_create_user_endpoint_not_present(api_client, user_data):
 @pytest.mark.django_db
 def test_regional_user_update_user_in_region(api_client, regional_user, user_in_region):
     """Test regional staff can update users in their region."""
-    assert regional_user.region.id == user_in_region.stt.region.id
+    assert user_in_region.stt.region in regional_user.regions.all()
```

Comment:
during review, I noticed the tests here for regional staff interactions with the user accounts in/out of their region. Its come to my attention that regional user group is currently configured to have visibility into user accounts, and its not just users in their region. These permissions need to be removed before onboarding of regional staff users. Currently no impact. will spin up new ticket, as these permissions were pre-existing  @elipe17 @lhuxraft